app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/webrtc/base/RtcCall.kt

```diff
@@ -161,46 +147,59 @@ abstract class RtcCall(protected  val client: RxWebSocketClient) {
     private fun createCapturer(isFacingFront: Boolean = false): CameraVideoCapturer? {
         val enumerator = Camera1Enumerator()
         for (name in enumerator.deviceNames) {
-            if (isFacingFront && enumerator.isFrontFacing(name)) {
-                return enumerator.createCapturer(name, null)
-            } else if (!isFacingFront && enumerator.isBackFacing(name)) {
+            if (isFront(isFacingFront, enumerator, name) || isBack(isFacingFront, enumerator, name)
+            ) {
                 return enumerator.createCapturer(name, null)
             }
         }
         return null
     }
 
+    private fun isBack(
+        isFacingFront: Boolean,
+        enumerator: Camera1Enumerator,
+        name: String?
+    ) = !isFacingFront && enumerator.isBackFacing(name)
+
+    private fun isFront(
+        isFacingFront: Boolean,
+        enumerator: Camera1Enumerator,
+        name: String?
+    ) = isFacingFront && enumerator.isFrontFacing(name)
+
     private fun handleMessage(jsonObject: JSONObject) {
         if (jsonObject.has(STATE_CHANGE_MESSAGE)) {
             Single.fromCallable {
                 val state = jsonObject.getString(STATE_CHANGE_MESSAGE)
                 this@RtcCall.remoteView?.setBackgroundColor(Color.TRANSPARENT)
                 state
             }.subscribeOn(AndroidSchedulers.mainThread())
-                    .subscribeBy(
-                            onSuccess = { Timber.i("Success $it") }
-                    )
+                .subscribeBy(
+                    onSuccess = { Timber.i("Success $it") }
+                )
         }
     }
 
     companion object {
         internal const val WEB_SOCKET_ACTION_KEY = "action"
 
-        internal const val WEB_SOCKET_ACTION_CALL = "call"
-        internal const val WEB_SOCKET_ACTION_RINGING = "ringing"
-        internal const val WEB_SOCKET_ACTION_CONNECTED = "connected"
-
         internal const val P2P_OFFER = "offerSDP"
         internal const val P2P_ANSWER = "answerSDP"
 
-        internal const val BABY_IS_CRYING = "BABY_IS_CRYING"
-        internal const val EVENT_RECEIVED_CONFIRMATION = "CRYING_EVENT_MESSAGE_RECEIVED"
-        internal const val PUSH_NOTIFICATIONS_KEY = "PUSH_NOTIFICATIONS_KEY"
+        internal const val ADD_FIREBASE_TOKEN = "ADD_FIREBASE_TOKEN"
```

Comment:
Will it be compatible with iOS?

Reply:
Right, this needs to be reverted in order to maintain iOS compatibility